### PR TITLE
Copter: fix metadata in FS_GCS_ENABLE param

### PR DESCRIPTION
**Pull request by Hwurzburg**
comment only change

### CODE REVIEW

**Comment by rmackay9** [comment]
The wording looks good to me.  It seems the parameter parser isn't happy though, "ValueError: Expected a colon seperator in ( see FS_OPTIONS))".  We probably just need to remove the comma before "see FS_OPTIONS"

**Comment by rmackay9** [comment]
@Hwurzburg,

Craig and I had a discussion about this in the weekly-devcall chat and we came up with this alternative wording:

// @Param: FS_GCS_ENABLE
// @Description: Action taken when connection with Ground station is lost for at least 5 seconds. See FS_OPTIONS param for exceptions when actions will not be taken
// @Values: 0:Disabled/NoAction,1:RTL,2:RTL or Continue with Mission in Auto (Deprecated in 4.0+ see FS_OPTIONS),3:SmartRTL or RTL,4:SmartRTL or Land,5:Land (4.0+ Only)

In fact, in my discussion with Craig we didn't have option 3 but we probably need it there.

What do you think?

File: ArduCopter/Parameters.cpp [hunk 1]
```diff
@@ -172,8 +172,8 @@ const AP_Param::Info Copter::var_info[] = {
 
     // @Param: FS_GCS_ENABLE
     // @DisplayName: Ground Station Failsafe Enable
-    // @Description: Controls whether failsafe will be invoked (and what action to take) when connection with Ground station is lost for at least 5 seconds. NB. The GCS Failsafe is only active when RC_OVERRIDE is being used to control the vehicle.
-    // @Values: 0:Disabled,1:Enabled always RTL,2:Enabled Continue with Mission in Auto Mode (Deprecated in 4.0+),3:Enabled always SmartRTL or RTL,4:Enabled always SmartRTL or Land,5:Enabled always land (4.0+ Only)
+    // @Description: Controls whether failsafe will be invoked (and what action to take) when connection with Ground station is lost for at least 5 seconds. See FS_OPTIONS param for additional actions, or for cases allowing Mission continuation, when GCS failsafe is enabled.
+    // @Values: 0:Disabled/NoAction,1:RTL,2:RTL or Continue with Mission in Auto Mode (Deprecated in 4.0+-same as bit 1),3:SmartRTL or RTL,4:SmartRTL or Land,5:Land (4.0+ Only)
```
Comment (rmackay9) [inline ArduCopter/Parameters.cpp:176]: Looking OK but I don't think that most users can do bit math... so I think we should just say "See FS_OPTIONS"

Comment (Hwurzburg) [reply ArduCopter/Parameters.cpp:176]: what about 2:RTL or Continue with Mission in Auto Mode ( in 4.0+- just RTL)

Comment (rmackay9) [reply ArduCopter/Parameters.cpp:176]: I think we should try and get users to stop using "2" though on 4.0+.  I think it should be:

2: RTL or Continue with Mission in Auto Mode (Deprecated in 4.0+ See FS_OPTIONS)

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**